frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/OverrideForkPointDialog.kt

```diff
@@ -79,13 +81,45 @@ class OverrideForkPointDialog(
             getString(
               "action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.radio-button.inferred"
             ),
-            radioButtonComment
+            branch.forkPoint?.shortHash ?: "cannot resolve commit hash"
           ),
           OverrideOption.INFERRED
         )
-          .comment(
-            thisRowComment
-          )
+          .comment(escapeHtml4(branch.forkPoint?.shortMessage) ?: "cannot resolve commit message")
+      }
+
+      row {
+        rb = radioButton(
+          format(
+            getString(
+              "action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.radio-button.custom"
+            ),
+            branch.forkPoint?.shortHash ?: "cannot resolve commit hash"
+          ),
+          OverrideOption.CUSTOM
+        )
+
+        val list = branch.commitsUntilParent.reverse() + listOf(branch.forkPoint, parent.pointedCommit)
```

Comment:
As discussed... shouldn't the parent be at top (==beginning of the list)? to match the top-to-down==oldest-to-latest ordering from status 🤔

Reply:
good point. check it now pls